### DISCUSSION TITLE
Honk notifying on a comment left on a PR without being attached to a review

### DISCUSSION THREAD

**Discussion by sfildis-gc** [oneroar/discussions#11] [Q&A] [answered]
Hi, we've recently noticed that Honk does not seem to honk with ordinary comments on a PR.

An example for this would be to create a PR, leave a normal comment using the box attached
<img width="928" alt="Screenshot 2024-01-17 at 10 07 51" src="https://github.com/oneroar/discussions/assets/84453230/44e9fdbe-c0da-47ae-bb80-22ebc438c4d4">

And the comment has not raised any notifications. 

PS the image is from an example PR which is not linked to Honk 😅 

Is this a problem with us or is it a bug? 

Thanks

**Comment by DragosDumitrache** [comment, marked as answer]
This is actually an expected behavior, comments on a PR are not part of a review, and they are not currently supported. However, this is not to say they won't be supported at a later date
